Jenkinsfile

```diff
@@ -66,20 +85,36 @@ spec:
                 container('docker') {
                     withCredentials([usernamePassword(credentialsId: 'argoGithub', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USERNAME')]) {
                         sh "git tag ${version}"
-                      sh "git push https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/icgc-argo/workflow-gateway --tags"
+                      sh "git push https://${GIT_USERNAME}:${GIT_PASSWORD}@github.com/${githubRepo} --tags"
```

Comment:
could you make `githubRepo` -> `gitHubRepo` just to follow our camel case convention.

I've made this change elsewhere as I am normalizing our params.

Reply:
Updated. Will stick with the camel-case moving forward.